### PR TITLE
modified greetings section and bottom navigation

### PR DESCRIPTION
**Pull request by Felix-Kariuki**

### CODE REVIEW

File: app/src/main/java/com/flexcode/github/sportifyui/ui/HomeScreen.kt [hunk 1]
```diff
@@ -127,9 +127,16 @@ fun Greetings() {
             verticalAlignment = Alignment.CenterVertically,
             modifier = Modifier.wrapContentWidth()
         ) {
+            Icon(
+                painter = painterResource(id = R.drawable.ic_notifications),
```
Comment (Felix-Kariuki) [inline app/src/main/java/com/flexcode/github/sportifyui/ui/HomeScreen.kt:131]: add badge to notification icon

**Review by ronnieotieno — COMMENTED** [review]
Hey, thanks for PR but your changes are out of the scope of your commit. 

1. There is no need to change the application id, was this a mistake?
2. Please don't change the settings on `gradle-wrapper.properties` or `build.gradle` files.

Make the changes then send a new commit.

**Comment by Carrieukie** [comment]
😂😂😂

**Comment by Felix-Kariuki** [comment]
😂😂

**Pull request closed without merging**